### PR TITLE
add gromacs wrappers

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

File: tools/gmx_md/md.xml [hunk 1]
```diff
@@ -0,0 +1,227 @@
+<tool id="protein" name="MD production simulation." version="0.1.0">
+    <requirements>
+        <requirement type="package">gromacs</requirement>
```
Comment (bgruening) [inline tools/gmx_md/md.xml]: version number is missing of gromacs

File: tools/gmx_minim/minim.xml [hunk 2]
```diff
@@ -0,0 +1,120 @@
+<tool id="protein" name="Energy minimization." version="0.1.0">
+    <requirements>
+        <requirement type="package">gromacs</requirement>
```
Comment (bgruening) [inline tools/gmx_minim/minim.xml]: version number

**Comment by bgruening** [comment]
Do not forget the shed.yml file, so we can see the tests here running. Thanks. Will be back in 6h.

**Comment by simonbray** [comment]
Sure, will do

**Comment by chrisbarnettster** [comment]
Hi @simonbray 

Looking good! 

Could you change the tool id, all id's are set to protein, perhaps gmx_minim, gmx_md and so on?

**Comment by simonbray** [comment]
Hi @chrisbarnettster, thanks for spotting that, I'll give them some better names.

**Comment by bgruening** [comment]
Super cool. Tests are running and have passed. Time to merge I would say. Before I do this I need to check travis and configure the repo accordingly. Will do so this evening.

Thanks @simonbray!

**Comment by simonbray** [comment]
Cool beans :)

File: tools/gromacs/npt.xml [hunk 3]
```diff
@@ -0,0 +1,180 @@
+<tool id="gmx_npt" name="Gromacs NPT" version="@VERSION@">
```
Comment (bgruening) [inline tools/gromacs/npt.xml]: please add for every tool a short description and make sure the `name` is also meaningful. The purpose of `name`+`description` is that in the Galaxy toolbox this is shown. Have a look at usegalaxy.eu how this is done usually. You have a short name, with a slightly longer description.

NPT is just not very intuitive. You probably also want to mention the name gromacs maybe?

File: tools/gromacs/nvt.xml [hunk 4]
```diff
@@ -0,0 +1,166 @@
+<tool id="gmx_nvt" name="NVT." version="@VERSION@">
```
Comment (bgruening) [inline tools/gromacs/nvt.xml]: here as well ... name and description please, so that a user can get a glimpse of what it is doing

**Comment by simonbray** [comment]
Hope this is better @bgruening?

**Comment by bgruening** [comment]
@chrisbarnettster @simonbray tools are installed on usegalaxy.eu

**Pull request merged**